cgo/lapack.go

```diff
@@ -871,6 +871,11 @@ func (impl Implementation) Dbdsqr(uplo blas.Uplo, n, ncvt, nru, ncc int, d, e, v
 func (impl Implementation) Dgebrd(m, n int, a []float64, lda int, d, e, tauQ, tauP, work []float64, lwork int) {
 	checkMatrix(m, n, a, lda)
 	minmn := min(m, n)
+	ws := max(m, n)
+	if lwork == -1 {
+		work[0] = float64(ws)
```

Comment:
If `m` and `n` are both 0, then the optimal work will also be 0. Could you please add something in respect here and in Dgebrd?